src/Differ/ListDiffer.php

```diff
@@ -58,7 +58,7 @@ public function __construct( $arrayComparer = null ) {
 	}
 
 	/**
-	 * @param $arrayComparer
+	 * @param ArrayComparer|int|null $arrayComparer one of the self::MODE_... constants
```

Comment:
This suggests to me it needs to be `one of the self::MODE_... constants`, which is not the case. This method is private and the class will likely not change for a long time, so perhaps putting time into this does not gain us much?